spec/lib/mercury/sync_spec.rb

```diff
@@ -7,10 +7,10 @@
   let!(:source) { 'test-exchange1' }
   let!(:queue) { 'test-queue1' }
   describe '::publish' do
-    %w{with without}.each do |w|
+    %w(with without).each do |w|
```

Comment:
I am ok with this rule about using parens instead of curly braces for `%w`, `%q`, etc..